CONTRIBUTING.md

```diff
@@ -57,21 +57,23 @@ We have several components
 
 ## Releasing
 
-We release through [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) which autoincrement using [Semantic versioning](https://semver.org/).
+Releases are done whenever the versionnumber gets bumped. For more information see the `Commit messages and versioning (Major, Minor, Patch updates)` section below.
 
 We use mooltiverse [Nyx](https://mooltiverse.github.io/nyx/guide/user/introduction/how-nyx-works/) for changelog and publishing to github.
 
 Run `./gradlew tasks --group Release` to see all release tasks
 
 Use `./gradlew nyxMake` to see what is build in [build/distributions](./build/distributions/).
 
-### Major, Minor, Patch updates
+### Commit messages and versioning (Major, Minor, Patch updates)
 
-Each commit with `!` just before the color `:` is a major update. So use it wisely. You can also add `BREAKING CHANGE:` in the long commit message format.
+Versionnumbers are updated according to [Semantic versioning](https://semver.org/), using [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/). Please be aware that only new releases will be done when one of the above prefixes is used. Other prefixes do not indicate user-facing changes and will therefore not result in a version bump, consequently not resulting in a new (pre) release. 
```

Comment:
Looking at the https://github.com/molgenis/molgenis-service-armadillo/blob/b3304cc1d4ba9ffb6cfcfd48876ea444af94b84a/CONTRIBUTING.md

Can we have white lines in these three sentences like below

```
Versionnumbers are updated according to [Semantic versioning](https://semver.org/), using [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/).

Please be aware that only new releases will be done when one of the above prefixes is used.

Other prefixes do not indicate user-facing changes and will therefore not result in a version bump, consequently not resulting in a new (
```